### DISCUSSION TITLE
Add `outline-color` to `transitionProperty`

### DISCUSSION THREAD

**Discussion by deebov** [tailwindlabs/tailwindcss#10384] [Ideas]
Hey!

I was wondering why it's not already included in `transitionProperty`.

Anyway, happy to create a PR for this!

**Comment by adamwathan** [comment]
Hey good catch! Have updated it here, will be in the next release 👍 

https://github.com/tailwindlabs/tailwindcss/pull/10385

Included you as a co-author on the commit, thanks!
